drivers/rs485_linux/work/arbiter/implem/default/CPP/src/arbiter_state.h

```diff
@@ -0,0 +1,27 @@
+// Fill in this class with your context data (internal state):
+// list all the variables you want global (per function instance)
+#include "dataview-uniq.h"
+
+#include <atomic>
+#include <chrono>
+
+#define QUEUE_SIZE 1000
+#define QUEUE_PACKETS_MAX_COUNT 10
+
+using namespace std::chrono_literals;
+
+class arbiter_state {
+public:
+    inline static const std::chrono::milliseconds listening_window_time = 100ms;
+    inline static const uint8_t open_listening_window_message[] = {
+        0x18, 0xbb, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0xff, 0x00, 0xaa, 0xc9, 0xf3
```

Comment:
Why this particular message?

Reply:
It is the shortes possible packet that holds 1 byte payload. The payload is 0xFF. As a sender and getter PID the values 0xAA and 0xBB are used taken from the darkest corner of imagination.